ShoppingCart.php

```diff
@@ -279,6 +281,34 @@ public function getCost($withDiscount = false)
     }
 
     /**
+     * Return full cart shipping cost
+     * @param $withDiscount
+     * @return int
+     */
+    public function getShippingCost($withDiscount = false)
```

Comment:
Actually I wrote two additional methods getShippingCost() and getTotal() in order to use something like this:
`$cart = \Yii::$app->cart;

$cart->attachBehavior('percentDiscount', ['class' => 'common\components\PercentDiscount', 'ammount' => 10]);

$cart->attachBehavior('fixedShippingCost', ['class' => 'common\components\FixedShippingCost', 'ammount' => 89, 'freeSince' => 999]);

$subtotal = \Yii::$app->cart->getCost(true);

$shipping = \Yii::$app->cart->getShippingCost();

$total = \Yii::$app->cart->getTotal(true);`

If there is a better way to get subtotal, shipping cost and total independently?